src/spclient.rs

```diff
@@ -56,21 +46,182 @@ pub enum OutputSpendStatus {
     Mined(MinedInBlock),
 }
 
+type WalletFingerprint = [u8; 8];
+
 #[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
 pub struct OwnedOutput {
-    pub txoutpoint: String,
     pub blockheight: u32,
     pub tweak: String,
```

Comment:
I'm wondering if we can make this `tweak` be a Scalar type. Initially we had these set to primitive types since those can be serialized easily, but I think it's probably better to implement custom de/serializers instead.

Reply:
I tinkered a bit with that, I think we better keep it for another PR that's not trivial